### ISSUE TITLE
Outgroups not added in align_selected_and_tree

### ISSUE THREAD

**Issue by joyceem** [chrisjackson-pellicle/ParaGone#6]
Hi Chris,

I'm having an issue with the `align_selected_and_tree` step. Up until then, everything runs fine, and I have all my cleaned sequences in the `09_sequences_from_qc_trees` (inlcuding all internal outgroup sequences). However, when I go to run `align_selected_and_tree` step, all of my internal outgroup sequences are removed. 

I can see from the log files that it identifies the internal outgroup sequences and the ones for which there are multiple copies (so it seems that the `filter_internal_outgroups` function works), but then it doesn't write any of them to my `10_sequences_from_qc_outgroups_added` fasta files. I can't figure out why this would be.

I did have to reformat the sequence fasta headers from the output of CAPTUS to match the HybPiper format, so I could have made a mistake there? But otherwise I really don't know what's going on. I've attached the log file and some example fasta files from the `09` and `10` folder - any help would be really appreciated!

Many thanks,
Lizzy
[cap_eg.zip](https://github.com/chrisjackson-pellicle/ParaGone/files/15466489/cap_eg.zip)

**Comment by chrisjackson-pellicle** [comment]
Hi Lizzy,

Ah, I think it's because you called the command with:

```
paragone align_selected_and_tree 09_sequences_from_qc_trees --pool 8 --threads 16 --trimal_gapthreshold 0.25 --generate_bootstraps
```

Instead of the folder `09_sequences_from_qc_trees`, this command [expects](https://github.com/chrisjackson-pellicle/ParaGone/wiki/Tutorial#step-4-phylogenetic-trees-from-selected-sequences) the folder `04_alignments_trimmed_hmmcleaned`, as it will extract any needed internal outgroup sequences from the alignment files in this folder.

I've made the same mistake - I'll insert a warning if any outgroup sequences can't be found in the next update.

Cheers,

Chris

**Comment by joyceem** [comment]
Ah, I see!! My silly mistake. Thanks a lot for your help Chris :D

Cheers,
Lizzy

**Comment by chrisjackson-pellicle** [comment]
No worries!

**Comment by chrisjackson-pellicle** [comment]
Fixed in [version 1.1.0](https://github.com/chrisjackson-pellicle/ParaGone/blob/main/change_log.md)
